cloud/amazon/iam_policy.py

```diff
@@ -27,54 +27,34 @@
     required: true
     default: null
     choices: [ "user", "group", "role"]
-    aliases: []
   iam_name:
     description:
       - Name of IAM resource you wish to target for policy actions. In other words, the user name, group name or role name.
     required: true
-    aliases: []
   policy_name:
     description:
       - The name label for the policy to create or remove.
-    required: false
-    aliases: []
+    required: true
   policy_document:
     description:
       - The path to the properly json formatted policy file (mutually exclusive with C(policy_json))
     required: false
-    aliases: []
   policy_json:
     description:
       - A properly json formatted policy as string (mutually exclusive with C(policy_document), see https://github.com/ansible/ansible/issues/7005#issuecomment-42894813 on how to use it properly)
     required: false
-    aliases: []
   state:
     description:
       - Whether to create or delete the IAM policy.
-    required: true
-    default: null
+    required: false
```

Comment:
I don't believe this is correct, as we have in the past stated that explicit is better than implicit when it comes to state parameters. I'm ok with correcting the above doc mistake, but I don't believe we should make this state implicit.